keyboards/elephant42/keymaps/default/keymap.c

```diff
@@ -0,0 +1,119 @@
+/* Copyright 2021 illness072
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#ifdef OLED_ENABLE
+#   include <stdio.h>
+#   include <string.h>
+#endif
+
+#define KC_VU KC__VOLUP
+#define KC_VD KC__VOLDOWN
+#define KC_MU KC__MUTE
```

Comment:
There are not the recommended volume controls.  They may not (will not) work on most systems.